### ISSUE TITLE
Write unit test for twisted.web.tap.Options.postOptions

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#7709]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)| @adiroiban reported|
|-|-|
|Trac ID|trac#7709|
|Type|enhancement|
|Created|2014-11-10 16:59:25Z|

As a follow up of [#7000](https://github.com/twisted/twisted/issues/7000)#comment:11

There is no test for the method and any changes introducing errors will not be detected.

<details><summary>Searchable metadata</summary>

```
trac-id__7709 7709
type__enhancement enhancement
reporter__adiroiban adiroiban
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__duplicate duplicate
component__web web
keywords__ 
time__1415638765504756 1415638765504756
changetime__1423223301757493 1423223301757493
version__None None
owner__adiroiban adiroiban

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_3"></a>|@adiroiban commented|
|-|-|

This is a blocker for [[#7000](https://github.com/twisted/twisted/issues/7000)](https://github.com/twisted/twisted/issues/7000) so once this is merge we should be able to also merge #7000

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_4"></a>|@adiroiban set status to `closed`|
|-|-|

I think that exarkun is right and this should be fixed in [#7000](https://github.com/twisted/twisted/issues/7000)
